lib/sidekiq_unique_jobs/sidekiq_unique_jobs.rb

```diff
@@ -188,7 +188,7 @@ def configure(options = {})
   # @return [String] a string like `5.0.2`
   #
   def fetch_redis_version
-    redis { |conn| conn.info("server")["redis_version"] }
+    Sidekiq.redis_info["redis_version"]
```

Comment:
Is this method available on all Sidekiq versions?

Reply:
since 4.1.
https://github.com/mperham/sidekiq/blame/b2a8d5bffe2d34fa0bef08f1e755aa5a2d484df5/lib/sidekiq.rb